fabtests/include/shared.h

```diff
@@ -254,7 +255,8 @@ extern int ft_socket_pair[2];
 extern int sock;
 extern int listen_sock;
 #define ADDR_OPTS "B:P:s:a:b::E::C:F:"
-#define FAB_OPTS "f:d:p:D:i:HK"
+#define FAB_OPTS "f:d:p:K"
+#define HMEM_OPTS "D:i:H"
 #define INFO_OPTS FAB_OPTS "e:M:"
```

Comment:
Should add HMEM_OPS to INFO_OPS to maintain the original functionality. A lot of tests use INFO_OPS for command line parsing.

Reply:
Thanks for the review, Jianxin. I fixed it in the latest change.